src/utilities/wellKnownDid/wellKnownDid.ts

```diff
@@ -88,10 +88,13 @@ export async function verifyDidConfigResource(
       }
 
       const { verified } = await DidUtils.verifyDidSignature({
-        keyId: issuerDidDetails.getKeyIds(KeyRelationship.assertionMethod)[0],
-        signature: credential.proof.signature as string,
+        signature: {
+          keyId: issuerDidDetails.getVerificationKeys(
+            KeyRelationship.assertionMethod,
+          )[0].id,
```

Comment:
I think the key ID here needs to be assembled, as this must be the key URI for a DID signature.

Reply:
Confusing naming again :) I’ve updated that.